packages/sdk/wallets/wallet-walletconnect/test/in-memory-wallet.ts

```diff
@@ -0,0 +1,145 @@
+import { CeloTx } from '@celo/connect'
+import { Address, newKit } from '@celo/contractkit'
+import { toChecksumAddress } from '@celo/utils/lib/address'
+import { EIP712TypedData } from '@celo/utils/lib/sign-typed-data-utils'
+import WalletConnect, { CLIENT_EVENTS } from '@walletconnect/client'
+import { PairingTypes, SessionTypes } from '@walletconnect/types'
+import { SupportedMethods } from '../src/types'
+
+const privateKey = '04f9d516be49bb44346ca040bdd2736d486bca868693c74d51d274ad92f61976'
+const kit = newKit('https://alfajores-forno.celo-testnet.org')
+kit.addAccount(privateKey)
+const wallet = kit.getWallet()!
+const [account] = wallet.getAccounts()
+
+export const testPrivateKey = privateKey
+export const testAddress = toChecksumAddress(account)
+
+// personal_sign is the one RPC that has [payload, from] rather
+// than [from, payload]
+function parsePersonalSign(req: any): { from: string; payload: string } {
```

Comment:
I'm gonna be that guy (and of course super optional), but what about using io-ts here :P

Reply:
Maybe the ship has sailed with io-ts + fp-ts in the SDKs but FYI using them adds ~250kb to the bundle size. So a hidden cost there is it moves us further away from a lean SDK :)

Reply:
Hmm, very great point